client/src/main/java/com/paypal/selion/reports/runtime/SaverFileSystem.java

```diff
@@ -131,8 +131,8 @@ public PageContents getScreenshotByName(String name) throws IOException {
     @Override
     public void init() {
         logger.entering();
-        (new File(outputFolder)).mkdirs();
```

Comment:
@fakrudeen78 the outputfolder points to target/surefire-reports/ which gets created by surefire plugin. It is used as the base directory to store all the details of reports. This statement can be removed because the outputfolder is already created.